lib/ansible/modules/lineinfile.py

```diff
@@ -87,13 +87,11 @@
       - If specified, the line will be inserted after the last match of specified regular expression.
       - If the first match is required, use(firstmatch=yes).
       - A special value is available; V(EOF) for inserting the line at the end of the file.
-      - If specified regular expression has no matches, EOF will be used instead.
+      - If specified regular expression has no matches or no value is passed, V(EOF) will be used instead.
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
hi @Akasurde ,

In that case, should we go back and add a changelog fragment for https://github.com/ansible/ansible/pull/83577 as well? It was merged without one.

In community.general there is no need for changelog fragments on PRs that only touch documentation and tests. Is it different in the ansible-core project? TIA

Reply:
NVM